pkgs/applications/virtualization/toolbox/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, go-md2man
+, installShellFiles
+, fetchpatch
+, glibc
+}:
+
+buildGoModule rec {
+  pname = "toolbox";
+  version = "0.0.99";
+
+  src = fetchFromGitHub {
+    owner = "containers";
+    repo = "toolbox";
+    rev = version;
+    sha256 = "CKjndWXDLxQ5epco/+TwRFxftbeMhYp/7NOWglF6EL8=";
+  };
+  modRoot = "src";
+
+  patches = [
+    ./toolbox_paths.patch
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/containers/toolbox/pull/675.patch";
```

Comment:
We usually prefer using the commits directly rather than the PR diff, e.g.

https://github.com/NixOS/nixpkgs/blob/9a92886922d507de4b34d98995ba3e19442624a2/pkgs/development/ruby-modules/bundix/default.nix#L21-L24

It's fine for now but it should be changed prior to merging.

Reply:
How do you do that again on github? I tried googling but I'm not getting anything.

Reply:
Nvm, just remembered. Updating now